### ISSUE TITLE
Some version 2.x releases can not be found

### ISSUE THREAD

**Issue by martinandersson** [renatoathaydes/setup-java-sdkman#2]
With version value `v2.x` as used in the examples in README.md, I get this error message from GitHub:

> Unable to resolve action `renatoathaydes/setup-java-sdkman@v2.x`, unable to find version `v2.x`

Same message for value `v2` as well. The one I did get working is `2.1.0`.

**Comment by renatoathaydes** [comment]
You can find all versions in the GitHub Release page, as with any GitHub Project: https://github.com/renatoathaydes/setup-java-sdkman/releases

**Comment by TarqusMiltonium** [comment]
Hi, I also had this issue - the need to specify a specific version is not at all mentioned in the README, but instead every example references the Action using "v2.x" -- which "clearly" indicates that we should do the same. Also, these same examples reference specific Java runtimes (as passed through to SDKman), indicating that specific versions are needed for those, but by implication not for this module. Also, since many other Actions simply use "v2", the seeming-lack of a specific version isn't quite as intuitive here as other usual contexts.

Worse, I _did_ guess that this might be the case, and tried using a specific tag in order to troubleshoot -- except v2.1.1 fails for me!  I only tried v2.1.0 after I read this Issue, which now works for me just fine.

You can see my successful/failed run attempts here:

v2.1.0: https://github.com/milton-apxium/run-rabbit-run/actions/runs/884756877
v2.1.1:  https://github.com/milton-apxium/run-rabbit-run/runs/2706646711?check_suite_focus=true

Commit to change versions: 
https://github.com/milton-apxium/run-rabbit-run/commit/8532a318656f3812386638c59b6160281c788995

So I guess this Issue should be reopened please, with a request to document the need to reference a specific version instead of "v2.x" -- an extra complete Example would be handy.

Thanks,
Milton.

**Comment by renatoathaydes** [comment]
Version 2.1.1 should be there! I will look into it.
Please make PRs with docs updates if you want them :) That's how open source works, right?

**Comment by renatoathaydes** [comment]
BTW just so you guys know: this project has zero stars... I assumed no one was using it, so I haven't really put any work on it since I got it working first time... would be nice to at least get some stars on this project to get an idea of whether people are using the project or not! I really don't care about having lots of stars, this is just a fork of the standard `setup-java` action... but when a project has zero stars, it really gives the impression no one cares about it :)

**Comment by TarqusMiltonium** [comment]
Yep, I can update the docs. I thought it was polite to discuss with the owner before forking & PR'ing? ;)

As for stars, I only stumbled across the project last Friday & was just trying it out - I didn't actually care about it until then! :D

**Comment by renatoathaydes** [comment]
Ok, thanks! Once the plugin works a bit better, I will "market" it (i.e. post about it on Reddit and Twitter) to get more "stars" :) I didn't do that yet as I didn't feel this project was ready... and I was right 😄
